views/ViewChart.react.js

```diff
@@ -1,19 +1,26 @@
 import React from 'react'
+import { connect } from 'react-redux'
 
 import { Typography } from '@material-ui/core'
 
 import AppLayout from './layouts/AppLayout'
+import BarGraph from './components/Graphs/BarGraph'
 
-const ViewChart = () => {
+const ViewChart = ({ graph }) => {
+  const { title, assessment, variable } = graph
   return (
     <AppLayout
-      title='View Chart'
+      title={variable.toUpperCase()}
```

Comment:
Should this be `title`?
```suggestion
      title={title.toUpperCase()}
```

Reply:
[e3318c6](https://github.com/AMP-SCZ/dpdash/pull/91/commits/e3318c65a03fb449fda9753002ed190da3a219fc)

After speaking with the client in the coming iterations we are adding things like description fields, etc... 
https://github.com/orgs/AMP-SCZ/projects/2#